src/client/components/localisation/style.css

```diff
@@ -42,8 +42,15 @@
 	border: 1px solid rgba(0, 0, 0, 0.4);
 }
 
+.localisation__canvasContainer {
+  flex-grow: 1;
+  position: relative;
+}
+
 .localisation__canvas {
-	flex-grow: 1;
+  width: 100%;
+  height: 100%;
+  position: absolute;
```

Comment:
When a child is position absolute and the parent is position relative, coordinates such as width and height become relative to the parent instead of the page.

Essentially the canvas is forced to take the shape of its parent, which flows nicely within the flexbox layout.

This gives us exactly what we want, an autosizing canvas element which flows with the natural page layout!